sys/net/include/ieee802154_frame.h

```diff
@@ -77,19 +113,11 @@ typedef enum __attribute__((packed)) {
  */
 #define LETOHS(a)   HTOLES(a)
 
+/**
+ * @brief   IEEE802.15.4 frame descriptor
+ */
 typedef struct __attribute__((packed)) {
-    ieee802154_frame_type_t frame_type;
-    uint8_t sec_enb;
-    uint8_t frame_pend;
-    uint8_t ack_req;
-    uint8_t panid_comp;
-    uint8_t dest_addr_m;
-    uint8_t frame_ver;
-    uint8_t src_addr_m;
-} ieee802154_frame_fcf_frame_t;
-
-typedef struct __attribute__((packed)) {
-    ieee802154_frame_fcf_frame_t fcf;
+    uint16_t fcf;
```

Comment:
maybe use `le_uint16_t` from (byteorder.h)[https://github.com/RIOT-OS/RIOT/blob/master/core/include/byteorder.h] for this and for the `*pan_id`s e.g. This way it's ensured they always will be little-endian.